Provide more information when adding evm accounts (#7219)
rotki/rotki: docs/api.rst

      {
        "result": {
            "success": {
               "eth": [
                  "0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B",
                  "0x19b0AD50E768D2376C6BA7de32F426ecE4e03e0b"
               ],
               "avax": ["0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B"],
               "optimism": ["0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B"]
            },
            "already_have_been_added": {
               "gnosis": ["0x19b0AD50E768D2376C6BA7de32F426ecE4e03e0b"]
            },
            "failed": {
               "avax": ["0x19b0AD50E768D2376C6BA7de32F426ecE4e03e0b"]
            },
         },
        "message": ""
      }

>    :resjson list success: A mapping containing the evm chain keys and which addresses were added for each chain.
>    :resjson list already_have_been_added: A mapping containing the evm chain keys and which addresses weren't added because they were added before.
>    :resjson list failed: A mapping containing the evm chain keys and which addresses failed to get added for each chain.

Reviewer: Just a suggestion for the names of the keys to be a bit more succint

```suggestion
   :resjson list added: A mapping containing the evm chain keys and which addresses were added for each chain.
   :resjson list existed: A mapping containing the evm chain keys and which addresses weren't added because they were added before.
   :resjson list failed: A mapping containing the evm chain keys and which addresses failed to get added for each chain.
```

      {
        "result": {
            "added": {
               "eth": [
                  "0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B",
                  "0x19b0AD50E768D2376C6BA7de32F426ecE4e03e0b"
               ],
               "avax": ["0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B"],
               "optimism": ["0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B"]
            },
            "existed": ["0x19b0AD50E768D2376C6BA7de32F426ecE4e03e0b"],
            "failed": {
               "avax": ["0x19b0AD50E768D2376C6BA7de32F426ecE4e03e0b"],
            },
            "no_activity": ["0x106B62Fdd27B748CF2Da3BacAB91a2CaBaeE6dCa"]
         },
        "message": ""
      }

   :resjson list added: A mapping containing the evm chain keys and which addresses were added for each chain.
   :resjson list existed: A list of addresses that weren't added to any new chain and already existed in the list of tracked addresses for some chain.

>    :resjson list failed: A mapping containing the evm chain keys and which addresses failed to get added for each chain.
>    :resjson list no_activity: A list of addresses without activity in any chain.

Reviewer: ```suggestion
   :resjson list failed: A mapping containing the evm chain keys and which addresses failed to get added for each chain due to some error contacting remote APIs.
   :resjson list no_activity: A list of addresses that were not added due to having no activity in any chain.
```
